examples/hf/t5/pippy_t5.py

```diff
@@ -154,52 +227,27 @@ def run_master(pp_ranks, args):
 
     torch.manual_seed(args.rank)
     inp = torch.empty(bs, seq_length, dtype=torch.long, device=device).random_(t5.config.vocab_size)
-    t5_input_dict = {'input_ids': inp, 'decoder_input_ids': inp,
-                     'labels': torch.empty(bs, seq_length, dtype=torch.long, device=device).random_(
-                         t5.config.vocab_size - 1)}
+
+    if args.mode == "train":
+        t5_input_dict = {'input_ids': inp, 'decoder_input_ids': inp,
+                        'labels': torch.empty(bs, seq_length, dtype=torch.long, device=device).random_(
+                            t5.config.vocab_size - 1)}
+    elif args.mode == "inference":
+        t5_input_dict = {'input_ids': inp, 'decoder_input_ids': inp}
```

Comment:
where do we load model checkpoints for inference?

Reply:
They are both using the same checkpoints, in this case its not a real checkpoint, it is mostly to demonstrate functionality other than that there is no real data going toward training/ inference.